distributed/scheduler.py

```diff
@@ -6257,7 +6259,10 @@ async def retire_workers(
                     )
                 if remove:
                     await asyncio.gather(
-                        *[self.remove_worker(address=w, safe=True) for w in worker_keys]
+                        *[
+                            self.remove_worker(address=w, safe=True, close=False)
+                            for w in worker_keys
+                        ]
```

Comment:
Why this change?

Reply:
This change adds `close=False` (default is True). This causes in most situations to issue a close_worker call twice since there is a dedicated close_worker call above. 

It _might_ be the correct thing to change this to `close=not close_workers` to avoid the duplication